crates/rolldown/tests/rolldown/function/inject/main.js

```diff
@@ -0,0 +1,8 @@
+import assert from 'node:assert'
+
+assert.strictEqual(Promise, 'promise-shim')
+assert.strictEqual(P, 'promise-shim')
+assert.strictEqual($, 'jquery')
+assert.strictEqual(fs.default, 'node-fs')
+// FIXME: oxc injects invalid statements `import { default as 'Object.assign' from 'object-assign-shim'` }`, so it fails.
```

Comment:
@Boshen cc

There is a bug here.

I take a rough look

oxc transform to the following code

```js
import { default as 'Object.assign'} from 'object-assign-shim'`
```

This is invalid js code.

while it should be 

```js
import { default as Object_assign} from 'object-assign-shim'`
```
or
```js
import Object_assign from 'object-assign-shim'`
```

Reply:
https://github.com/oxc-project/oxc/issues/4818